packages/framework/src/Model/Pricing/Rounding.php

```diff
@@ -36,19 +36,21 @@ public function roundPriceWithVatByCurrency(Money $priceWithVat, Currency $curre
 
     /**
      * @param \Shopsys\FrameworkBundle\Component\Money\Money $priceWithoutVat
+     * @param \Shopsys\FrameworkBundle\Model\Pricing\Currency\Currency $currency
      * @return \Shopsys\FrameworkBundle\Component\Money\Money
      */
-    public function roundPriceWithoutVat(Money $priceWithoutVat): Money
+    public function roundPriceWithoutVat(Money $priceWithoutVat, Currency $currency): Money
     {
-        return $priceWithoutVat->round(2);
+        return $priceWithoutVat->round($currency->getRoundingPlacesPriceWithoutVat());
     }
 
     /**
      * @param \Shopsys\FrameworkBundle\Component\Money\Money $vatAmount
+     * @param \Shopsys\FrameworkBundle\Model\Pricing\Currency\Currency $currency
      * @return \Shopsys\FrameworkBundle\Component\Money\Money
      */
-    public function roundVatAmount(Money $vatAmount): Money
+    public function roundVatAmount(Money $vatAmount, Currency $currency): Money
     {
-        return $vatAmount->round(2);
+        return $vatAmount->round($currency->getRoundingPlacesPriceWithoutVat());
```

Comment:
I am wondering whether this should be mentioned in the setting itself or it is an implementation detail - my point is that the admin thinks he sets the places for rounding the price with vat but internally, he also sets the places for VAT rounding, right?

Reply:
The setting is for price without VAT. I felt logical, that if this price can have 10 decimal places, then VAT amount should have same setting. Do you disagree?